core/eolearn/core/core_tasks.py

```diff
@@ -555,6 +557,32 @@ def map_method(self, feature):
         return feature[..., self.bands]
 
 
+class ExplodeBandsTask(EOTask):
+    """Explode a subset of bands from one feature to multiple new features."""
+
+    def __init__(
+        self,
+        input_feature: Tuple[FeatureType, str],
+        output_features: Iterable[Tuple[FeatureType, str]],
+        bands: Iterable[List[int]],
```

Comment:
another notice here, what do you do if the list of bands to extract is empty? do we delegate handling to `ExtractBandTask`?